### PR TITLE
#2604 ensure score sent to LMS before status

### PR DESCRIPTION
**Pull request by chris-steele**
Fixes https://github.com/adaptlearning/adapt_framework/issues/2604

See https://github.com/adaptlearning/adapt-contrib-core/pull/30

### CODE REVIEW

**Comment by chris-steele** [comment]
See https://github.com/adaptlearning/adapt-contrib-core/pull/30

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-spoor",
-  "version": "3.10.1",
+  "version": "3.10.2",
   "framework": ">=5.5",
```
Comment (cahirodoherty-learningpool) [inline bower.json:4]: Hi Chris, You will need to bump the FW dependency to include your Core additions

Comment (chris-steele) [reply bower.json:4]: Thanks @cahirodoherty-learningpool, should I bump package.json in adapt-contrib-core to 6.0.5 and then bump the FW dependency here to v5.17.7?

Comment (oliverfoster) [reply bower.json:4]: And let @joe-allen-89 know so he's aware for releases

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by oliverfoster — APPROVED** [review]

**Review by eleanor-heath — APPROVED** [review]

**Pull request merged**